htdocs/modules/system/blocks/system_blocks.php

```diff
@@ -179,6 +179,10 @@ function b_system_user_show()
     $block['lang_logout']        = _MB_SYSTEM_LOUT;
     $criteria                    = new CriteriaCompo(new Criteria('read_msg', 0));
     $criteria->add(new Criteria('to_userid', $xoopsUser->getVar('uid')));
+    // BigKev73 - Added to prevent MSGs that were deleted, but never read from showing up in the unread message count. I had several test messages that I batch delete from
```

Comment:
For future consideration, the in-depth commentary is appropriate in the commit message, but is distracting in the code itself. If anyone needs to know who changed what when and why, the git history is easily accessible.